django_classified/admin.py

```diff
@@ -33,9 +33,15 @@ class AreaAdmin(admin.ModelAdmin):
         'title',
     )
     prepopulated_fields = {'slug': ('title',)}
+    
+class ProfileAdmin(admin.ModelAdmin):
+    list_display = ('phone',)
```

Comment:
I think this will only display `phone` in the Profile list. I think it may be better to also display user here.
```
list_display = ('user', 'phone')
```

Reply:
list_display and search_fields have been updated so that the `username` and `phone` are displayed in the list and both fields are searchable.